spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/support/ArgumentSanitizer.java

```diff
@@ -18,13 +18,17 @@
 
 import java.util.regex.Pattern;
 
+import org.springframework.cloud.dataflow.core.dsl.ArgumentNode;
+
 /**
  * Sanitizes potentially sensitive keys for a specific command line arg.
  * @author Glenn Renfro
  */
 public class ArgumentSanitizer {
```

Comment:
I saw their implementation.  But it is not accessible.  So this is based on their implementation.